incubator/gitea/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "gitea.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
+{{- end }}
+spec:
+  type: {{ .Values.serviceType }}
+  ports:
+  - port: {{ .Values.service.httpPort | int }}
```

Comment:
In some files the list indentation is 0 and in other files it's 2 spaces. Can you please make it consistent across the chart at a 2 space indentation.